provisioner/ibm-s3fs-provisioner_test.go

```diff
@@ -603,6 +609,21 @@ func Test_Provision_APIKeyWithoutServiceInstanceIDInBucketCreation(t *testing.T)
 	}
 }
 
+func Test_Provision_PVCNamespaceNotAllowedInSecrets(t *testing.T) {
+	p := getCustomProvisioner(
+		&clientGoConfig{withAllowedNamespace: true},
+		&fake.ObjectStorageSessionFactory{},
+		uuid.NewCryptoGenerator(),
+	)
+	v := getVolumeOptions()
+	v.PVC.Annotations[annotationAutoCreateBucket] = "true"
```

Comment:
This line for setting `auto-create-bucket: true` is not required bcoz we are checking allowed NS stuff irrespective of value of flag `auto-create-bucket`.